cengage/react-magma
# fix gatsby empty image alts

Issue: #1349

## What I did
replace empty alts with "GATSBY_EMPTY_ALT"

## Screenshots:
![image](https://github.com/user-attachments/assets/12cc64f3-b94f-4760-8bc3-c9cb6d1fd12b)

## Checklist 

## How to test
navigate to a page that contains decorative images with empty alts like "/design/button/" and check for image alts
